tests/console/validate_activation_encrypted_partition.pm

```diff
@@ -21,10 +21,11 @@ use base "installbasetest";
 use scheduler 'get_test_suite_data';
 use testapi;
 use validate_encrypt_utils;
+use Utils::Backends 'use_ssh_serial_console';
 
 sub run {
     my $test_data = get_test_suite_data();
-    select_console 'install-shell';
+    check_var('BACKEND', 'pvm_hmc') ? use_ssh_serial_console : select_console 'install-shell';
```

Comment:
Hmm, I have issues with `use_ssh_serial_console` on powerVM, even though it already contains all the logic which console to select for which backend. But we will have to wait till we get powerVM workers back =(

Reply:
It worked in some test run I did before it breaks, But I did not note the url of that run :(

Reply:
From the code I also believe that it should work, because `use_ssh_serial_console` calls different shell depending on the config. So let's see when we get workers back.

Reply:
So, this should work fine now, so we don't need special condition for power VM here.